### ISSUE TITLE
genshi benchmark fails on Python 3.8.0a4+

### ISSUE THREAD

**Issue by vstinner** [python/pyperformance#54]
```
  File "lib/python3.8/site-packages/genshi/template/text.py", line 137, in __init__
    Template.__init__(self, source, filepath=filepath, filename=filename,
  File "lib/python3.8/site-packages/genshi/template/base.py", line 418, in __init__
    self._stream = self._parse(source, encoding)
  File "lib/python3.8/site-packages/genshi/template/text.py", line 181, in _parse
    for kind, data, pos in interpolate(text, self.filepath, lineno,
  File "lib/python3.8/site-packages/genshi/template/interpolation.py", line 77, in interpolate
    expr = Expression(chunk.strip(), pos[0], pos[1],
  File "lib/python3.8/site-packages/genshi/template/eval.py", line 93, in __init__
    self.code = _compile(node, self.source, mode=self.mode,
  File "lib/python3.8/site-packages/genshi/template/eval.py", line 470, in _compile
    return build_code_chunk(code, filename, name, lineno)
  File "lib/python3.8/site-packages/genshi/compat.py", line 94, in build_code_chunk
    return CodeType(0, code.co_nlocals, code.co_kwonlyargcount,
an integer is required (got type bytes)
```

**Comment by vstinner** [comment]
cc @pablogsal

**Comment by vstinner** [comment]
Bug fixed in Genshi master branch, wait for a release: https://github.com/edgewall/genshi/pull/19

**Comment by lazka** [comment]
Looks like this was fixed in cf0f71bd7511a624 and this can be closed.

**Comment by vstinner** [comment]
Oops right, I close the issue.
